build.sbt

```diff
@@ -61,7 +81,19 @@ lazy val activityStreamImpl = project("activity-stream-impl")
   .enablePlugins(LagomJava)
   .settings(
     version := "1.0-SNAPSHOT",
-    libraryDependencies += lagomJavadslTestKit
+    dockerRepository := Some("chirper"),
+    dockerUpdateLatest := true,
+    dockerEntrypoint ++= """-Dplay.crypto.secret="${APPLICATION_SECRET:-none}" -Dplay.akka.actor-system="${AKKA_ACTOR_SYSTEM_NAME:-activityservice-v1}" -Dhttp.address="$ACTIVITYSERVICE_BIND_IP" -Dhttp.port="$ACTIVITYSERVICE_BIND_PORT" -Dakka.actor.provider=cluster -Dakka.remote.netty.tcp.hostname="$(eval "echo $AKKA_REMOTING_BIND_HOST")" -Dakka.remote.netty.tcp.port="$AKKA_REMOTING_BIND_PORT" -Dakka.cluster.seed-nodes.0="akka.tcp://${AKKA_ACTOR_SYSTEM_NAME}@${AKKA_SEED_NODE_HOST}:${AKKA_SEED_NODE_PORT}" -Dakka.io.dns.resolver=async-dns -Dakka.io.dns.async-dns.resolve-srv=true -Dakka.io.dns.async-dns.resolv-conf=on""".split(" ").toSeq,
+    dockerCommands :=
+      dockerCommands.value.flatMap {
+        case ExecCmd("ENTRYPOINT", args @ _*) => Seq(Cmd("ENTRYPOINT", args.mkString(" ")))
+        case v => Seq(v)
+      },
+    resolvers += bintrayRepo("hajile", "maven"),
+    libraryDependencies ++= Seq(
+      lagomJavadslCluster,
```

Comment:
The `activity-stream-impl` does depend on the libraries defined by the `lagomJavadslCluster` since without this, the `activity-stream-impl` won't start and failed with unable to find the `ClusterRefActorProvider` class. 

When building to ConductR, this library dependency is being added as part of `sbt-conductr`.

Now that we may deploy to K8, and the `sbt-conductr` is not enabled when deploying to K8, we need to explicitly declare `lagomJavaDslCluster`.

Hopefully that makes sense.